### DISCUSSION TITLE
能开源libxdl.so吗？

### DISCUSSION THREAD

**Discussion by StaringTheWorldAtMyRearView123** [hexhacking/xUnwind#8] [General]
如题，想学习api的原理

**Comment by caikelun** [comment]
xDL一直是开源的：https://github.com/hexhacking/xDL

**Reply by StaringTheWorldAtMyRearView123** [reply]
感谢作者回复，请问基于Cfi和基于Eh 这两者的backtrace是什么？我对基于FP的backtrace已经很了解了，但CFI和EH不太清楚.

**Reply by StaringTheWorldAtMyRearView123** [reply]
或这么问，如果不提供FP寄存器, 那么 cfi 和 eh 这两者, 还能正常backtrace吗？
